Traducir.Api/Controllers/AccountController.cs

```diff
@@ -69,14 +69,19 @@ public async Task<IActionResult> OauthCallback(string code, string state = null)
                 return Content($"You need at least {minRep} to log in");
             }
 
-            await _userService.UpsertUserAsync(new User
+            // For development, only execute this if UPSERT_USER: true
+            if (_configuration.GetValue<string>("ASPNETCORE_ENVIRONMENT") != "Development" ||
+                _configuration.GetValue<bool>("UPSERT_USER"))
```

Comment:
Instead of having the default be not upserting, I’d do it the other way.

don’t checks the environment and check a new variable that is DO_NOT_UPSERT, so that people that want that behavior can set it up... but that by default, it performs the upsert.